images/nginx_php71/sites/controlpanel/index.php

```diff
@@ -18,14 +18,16 @@
     <h2>Other servers</h2>
     <ul>
         <li><a href="/memcache/">Memcached Info</a></li>
-        <li><a href="/phpmemadmin/web/index.php">Memcached Admin</a></li>
+        <li><a href="http://admin:pass@localhost:88/phpmemadmin/web/index.php">Memcached Admin</a></li>
         <li><a href="http://<?php echo $_SERVER['SERVER_NAME']; ?>:8983/solr/">Solr</a></li>
         <li><a href="/pma/index.php">PhpMyAdmin</a></li>
-        <li><a href="/va/html/">Varnish</a></li>
+        <li><a href="http://varnish:CacheMeIfYouCan@localhost:88/va/html/">Varnish</a></li>
     </ul>
 
-    <p>credentials for Varnish: varnish / CacheMeIfYouCan</p>
-    <p>credentials for Memcached Admin: admin / pass</p>
-
+    <p>Sometimes you might be asked for a password, then they are the following :</p>
```

Comment:
@ScreamZ Please replace with: 
"Login & password for Varnish & Memcached Admin are already present in the above links.
 If needed, here are the credentials: "